confluent_kafka/schema_registry/json_schema.py

```diff
@@ -243,7 +243,7 @@ class JsonDeserializer(Deserializer):
     """
     __slots__ = ['_parsed_schema', '_registry', '_from_dict']
 
-    def __init__(self, schema_registry_client, schema_str, from_dict=None):
+    def __init__(self, schema_str, from_dict=None, schema_registry_client=None):
```

Comment:
if you're changing the order like this, just remove the `schema_registry` parameter altogether - it can be added later without breaking backwards compatibility if needed (which it probably won't be).

Reply:
agreed

Reply:
poof

Reply:
If we change the order here should we also change the order on AvroDeserializer?

Reply:
it's gone entirely